inkcpp_cl/CMakeLists.txt

```diff
@@ -4,5 +4,16 @@ add_executable(inkcpp_cl inkcpp_cl.cpp test.h test.cpp)
 # Include compiler and runtime libraries
 target_link_libraries(inkcpp_cl PUBLIC inkcpp inkcpp_compiler)
 
+# For https://en.cppreference.com/w/cpp/filesystem#Notes
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "9.1")
+    target_link_libraries(inkcpp_cl PUBLIC stdc++fs)
```

Comment:
Why is this public only for GNU and not for clang?

Reply:
Sorry, no reason, now it is all private.